docker/llm/finetune/lora/README.md

```diff
@@ -53,3 +53,40 @@ cat launcher.log # display logs collected from other workers
 ```
 
 From the log, you can see whether finetuning process has been invoked successfully in all MPI worker pods, and a progress bar with finetuning speed and estimated time will be showed after some data preprocessing steps (this may take quiet a while).
+
+
+## To run in TDX-CoCo and enable Remote Attestation API
+
+You can deploy this workload in TDX CoCo and enable Remote Attestation API Serving with the referance yaml configuration `./kubernetes/bigdl-lora-finetuning-job.yaml.tdx`. The main diffences are it's need to execute the pods as root and mount TDX device, and a flask service is responsible for generating launcher's quote and collecting workers' quotes. 
```

Comment:
how to turn from normal mode to tdx mode when installing through helm? manually delete, move or rename yamls?

Reply:
Need to manually delete and rename the yaml